GameDefinition/PLAYING.md

```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
 of exploration.
 
 
+Managing inventory and crafting
+-------------------------------
+
+All items belonging to your team are stored either in the character's local
+inventory (called the character's outfit) or in the shared inventory that
+is accessible by every team member (called the team stash). Items that are
+in the character's outfit can be accessed by or are beneficial for this
+particular character only. Stashed items can be accessed by the team members
+(unless the character is injured or panicking). But no one has a direct
```

Comment:
To be more precise "is not calm enough" or "is stressed out", "e.g., due to being injured or panicking". This is clearly visible in the Calm field in the map UI (and the `#` menu). Also, this only affects the equipment, e.g., putting items from stash for outfit. You can still move item from stash to floor even totally panicked.

Reply:
I'll update. To my memory, I couldn't work with the stashed items when the calmness level was very low. I'll check (and rephrase the text).

Reply:
Seems to be so. Like the character with the calmness level of 0 cannot take anything from the stash, but can drop it from there.

Reply:
Yes, I think such character can also fling and trigger items from stash. Not from equipment, though.